pallets/automation-time/src/benchmarking.rs

```diff
@@ -229,19 +237,23 @@ benchmarks! {
 			times.push(hour);
 		}
 
-		let task_id: T::Hash = schedule_notify_tasks::<T>(caller.clone(), times.clone(), T::MaxTasksPerSlot::get() - 1);
-		let provided_id: Vec<u8> = vec![(T::MaxTasksPerSlot::get()/256).try_into().unwrap(), (T::MaxTasksPerSlot::get()%256).try_into().unwrap()];
+		let mut provided_id = schedule_notify_tasks::<T>(caller.clone(), times.clone(), T::MaxTasksPerSlot::get() - 1);
+		provided_id = increment_provided_id(provided_id);
 	}: schedule_native_transfer_task(RawOrigin::Signed(caller), provided_id, times, recipient, transfer_amount)
 
 	schedule_auto_compound_delegated_stake_task_full {
+		let max_tasks_per_slot: u32 = (
```

Comment:
is this the one benchmark using `T::MaxWeightPerSlot` instead of `T::MaxTasksPerSlot` because it weighs so much it fails on  `T::MaxTasksPerSlot`?

Reply:
Yup, exactly.